tests/foreman/ui/test_myaccount.py

```diff
@@ -1,171 +1,284 @@
 # -*- encoding: utf-8 -*-
 """Test class for Users UI
 
-@Requirement: Myaccount
+:Requirement: Myaccount
```

Comment:
@renzon in 6.2.z we use `@` as the token. you can call https://github.com/SatelliteQE/robottelo/blob/6.2.z/scripts/token_editor.py to fix it